### PR TITLE
Data Conventions Guidelines links failure.

### PR DESCRIPTION
**Pull request by wu-sheng**
In Tags section, ref of `Data Conventions Guidelines` is error, can't link to the page

### CODE REVIEW

File: _docs/pages/spec.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ While it is not a formal requirement, specific event names should apply to many
 
 Every Span may also have zero or more key:value **Tags**, which do not have timestamps and simply annotate the spans.
 
-As is the case with Logs, if certain known tag key:values are used for common application scenarios, tracers can choose to pay special attention to them. For more information, see the [Data Semantics Guidelines](api/data-conventions).
+As is the case with Logs, if certain known tag key:values are used for common application scenarios, tracers can choose to pay special attention to them. For more information, see the [Data Conventions Guidelines](api/data-conventions.html).
```
Comment (bensigelman) [inline _docs/pages/spec.md]: Let's do `/pages/api/data-conventions` (without the `.html`)

**Comment by wu-sheng** [comment]
@bensigelman , I remove the .html, and change the name of Guidlines

**Comment by yurishkuro** [comment]
lgtm

**Comment by bensigelman** [comment]
@wu-sheng thanks

**Pull request merged**